### ISSUE TITLE
How to predict a single unlabeled video?

### ISSUE THREAD

**Issue by mrlihellohorld** [Alvin-Zeng/PGCN#50]
Dear author, I have trained the PGCN model on my own data set, but I need to make a prediction on a video (not in the training set nor in the test set),  I see that the code needs to generate the corresponding proposal which need corresponding GT information, But the videos I'm testing now don't have tag files. could u tell me how to do it? thanks a lot

**Comment by hongkyhao** [comment]
Can I ask about how you trained the PGCN model on your own dataset? What kind of proposal list did you use on your dataset?

**Comment by mrlihellohorld** [comment]
> Can I ask about how you trained the PGCN model on your own dataset? What kind of proposal list did you use on your dataset?

自己的数据集就自己写一个脚本，按照ssn的proposal格式，有max_iou，max_overlap需要计算，其他都不用；就写个脚本然后算一下，按照ssn的格式生成就行了。

**Comment by hongkyhao** [comment]
mrlilhellohorld, can you give me your email? I want to ask several things on how to train PGCN on custom dataset. It will be very much appreciated and helped me a lot. Thank you

**Comment by mrlihellohorld** [comment]
> mrlilhellohorld, can you give me your email? I want to ask several things on how to train PGCN on custom dataset. It will be very much appreciated and helped me a lot. Thank you

2013101105@cuit.edu.cn

**Comment by mrlihellohorld** [comment]
The ground truth of video files is not needed for infering
